### PR TITLE
SAK-45158 rWiki fixing background and text visibility in dark mode

### PR DESCRIPTION
**Pull request by wyu12**
Screenshots for before and after are available on Jira: https://jira.sakaiproject.org/browse/SAK-45158

This doesn't fix all of the hardcoded values, but fixes dark mode readability for crucial components

### CODE REVIEW

File: rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css [hunk 1]
```diff
@@ -526,46 +527,66 @@ MOOT
...
 	-moz-border-radius: 0 0 10px 10px;
 	padding: 0 .5em;
 	width: auto;
-	background: #ddd;
+	background: var(--sakai-background-color-2);
 	padding: 0  10px  10px  10px
 }
-.rwikicomenttext p.paragraph{ margin: 0; padding: 0 }
+.rwikicomenttext p.paragraph{
+	background-color: var(--sakai-background-color-2);
```
Comment (wyu12) [inline rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css:542]: Not sure why Codacy flags this line - my code editor is set to 1 tab = 2 spaces and all the other indentation seems fine.

Comment (wyu12) [reply rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css:542]: @ern @profmikegreene

File: rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css [hunk 2]
```diff
@@ -53,7 +53,7 @@ ul.tabs li.tabHeadOff {
 	border-left: 1px solid #ccc;
 	border-top: 1px solid #ccc;
 	border-right: 1px solid #ccc;
-	background-color: #eee
+  background-color: var(--sakai-background-color-2);
```
Comment (profmikegreene) [inline rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css:56]: Please change spacing to tab to match the rest of this section of the file.

File: rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css [hunk 3]
```diff
@@ -67,7 +67,7 @@ ul.tabs li.tabHeadOn  {
 	border-left: 1px solid #ccc;
 	border-top: 1px solid #ccc;
 	border-right: 1px solid #ccc;
-	background-color: #fff
+  background-color: var(sakai-background-color-1);
```
Comment (profmikegreene) [inline rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css:70]: Please change spacing to tab to match the rest of this section of the file.

File: rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css [hunk 4]
```diff
@@ -148,23 +148,23 @@ a.editToolBar:hover,a.editToolBar:active,a.previewToolBar:hover,a.previewToolBar
...
+/* contains the on off switchers */
 #rwiki_sidebar_switcher{ padding-right: 4px }
 #rwiki_sidebar_switcher a{ padding-top: 4px }
 /* this causes the content area to expand to 100% */
 .withsidebar{
 	float: left;
 	width: 80%;
 	padding: 5px 0px 2px 0;
-	background: #fff
+	background: var(sakai-background-color-1);
```
Comment (profmikegreene) [inline rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css:167]:
```suggestion
	background: var(--sakai-background-color-1);
```

File: rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css [hunk 5]
```diff
@@ -187,7 +187,7 @@ a.anchorpoint { vertical-align: top }
 #restoreVersion, #restoreDate{ color: #777; background: transparent }
 #restoreDate{ width: 30em }
 #rwiki_content textarea:focus { border: 1px solid #777 }
-.rwiki_edit_wrapper{ background: #fff; height: 100%; overflow: hidden }
+.rwiki_edit_wrapper{ background: var(sakai-background-color-1); height: 100%; overflow: hidden }
```
Comment (profmikegreene) [inline rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css:190]:
```suggestion
.rwiki_edit_wrapper{ background: var(--sakai-background-color-1); height: 100%; overflow: hidden }
```

File: rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css [hunk 6]
```diff
@@ -526,46 +527,66 @@ MOOT
 	-webkit-border-top-right-radius: 10px;
 	-moz-border-radius: 10px 10px 0 0;
 	padding: .5em;
-	background: #ddd
+	background: var(--sakai-background-color-2)
```
Comment (profmikegreene) [inline rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css:530]:
```suggestion
	background: var(--sakai-background-color-2);
```

File: rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css (hunk 1 above)
Comment (profmikegreene) [reply rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css:542]: I believe there's a rule for adding a space before the `{` but I don't know if that's the issue. I wouldn't worry about Codacy in this instance.

File: rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css [hunk 7]
```diff
@@ -526,46 +527,66 @@ MOOT
...
 #createCommentLink{
 	background: url(/library/image/silk/comment_add.png) center left no-repeat;
 	padding:3em 1.6em;
 }
 
 .rwiki_comments { list-style: none; margin: 0; padding: 0 }
-.rwiki_comments .hidden .rwikicommenttop, .rwiki_comments .hidden .rwikicomenttext{background:#eee;color:#999}
+.rwiki_comments .hidden .rwikicommenttop, .rwiki_comments .hidden .rwikicomenttext{
+	background:var(--sakai-background-color-2);
+	color:var(--sakai-text-color-1)
```
Comment (profmikegreene) [inline rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css:586]:
```suggestion
	color: var(--sakai-text-color-1);
```

File: rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css [hunk 8]
```diff
@@ -526,46 +527,66 @@ MOOT
...
+}
 .rwiki_comments .hidden .commentAuthor{ /*.commentHeader*/
-	color: #bbb 
+	color: var(--sakai-text-color-2)
```
Comment (profmikegreene) [inline rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css:589]:
```suggestion
	color: var(--sakai-text-color-2);
```

File: rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css [hunk 9]
```diff
@@ -630,14 +651,16 @@ added only to site index, could be added to other things*/
 }
 #selectList{
 	list-style:none;
-	margin:0;padding:0
+	margin:0;
+	padding:0
```
Comment (profmikegreene) [inline rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css:655]:
```suggestion
	padding: 0;
```

**Review by profmikegreene — CHANGES_REQUESTED** [review]

**Review by ern — APPROVED** [review]

**Pull request merged**